kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -0,0 +1,260 @@
+{
+    "administration": {
+        "category": "administration",
+        "tags": [
+            "association_or_organization",
+            "city_hall",
+            "immigration_and_naturalization_service"
+        ]
+    },
+    "airport": {
+        "category": "airport",
+        "tags": [
+            "international_airport",
+            "airport"
+        ]
+    },
+    "artCulture": {
+        "category": "artCulture",
+        "tags": [
+            "art_gallery",
+            "art_museum",
+            "ballet_theater",
+            "concert_hall",
+            "conference_center",
+            "convention_center",
+            "cultural_center",
+            "historical_landmark",
+            "modern_art_museum",
+            "museum",
+            "opera_house",
+            "performing_arts_theater"
+        ]
+    },
+    "automobile": {
+        "category": "automobile",
+        "tags": [
+            "gas_station",
+            "park",
+            "school"
+        ]
+    },
+    "beauty": {
+        "category": "beauty",
+        "tags": [
+            "hair_salon"
+        ]
+    },
+    "cafe": {
+        "category": "cafe",
+        "tags": [
+            "cafe",
+            "coffee_shop"
+        ]
+    },
+    "drinks": {
+        "category": "drinks",
+        "tags": [
+            "bar",
+            "pub"
+        ]
+    },
+    "education": {
+        "category": "education",
+        "tags": [
+            "library",
+            "school"
+        ]
+    },
+    "entertainment": {
+        "category": "entertainment",
+        "tags": [
+            "banquet_hall",
+            "casino",
+            "concert_hall",
+            "golf_club",
+            "golf_course",
+            "hotel",
+            "live_music_venue",
+            "night_club",
+            "school",
+            "stadium"
+        ]
+    },
+    "food": {
+        "category": "food",
+        "tags": [
+            "banquet_hall",
+            "lunch_restaurant",
+            "restaurant"
+        ]
+    },
+    "groceries": {
+        "category": "groceries",
+        "tags": [
+            "grocery_store",
+            "supermarket"
+        ]
+    },
+    "medical": {
+        "category": "medical",
+        "tags": [
+            "hospital",
+            "medical_center"
+        ]
+    },
+    "misc": {
+        "category": "misc",
+        "tags": [
+            "attraction",
+            "bar",
+            "cafe",
```

Comment:
Ideally, tags should be only attributed to one high-level category. Especially with "misc", since it's more a placeholder category. So if something is manually mapped from `misc` to another category, the tag has to be removed from `misc` (e.g., "bar" and "cafe").

Reply:
'misc' now has less tags listed. no more overlapping with other kuwala tags.